pkg/controller/control.go

```diff
@@ -713,21 +714,37 @@ func (c *Controller) ReadAt(b []byte, off int64) (int, error) {
 	return n, err
 }
 
+func isSnapshotDiskExist(err error) bool {
+	match, _ := regexp.MatchString("snapshot (.*) is already existing", err.Error())
+	return match
+}
+
 func (c *Controller) handleErrorNoLock(err error) error {
 	if bErr, ok := err.(*BackendError); ok {
+		snapshotDiskIsExist := false
+
 		if len(bErr.Errors) > 0 {
-			for address, replicaErr := range bErr.Errors {
-				logrus.Errorf("Setting replica %s to ERR due to: %v", address, replicaErr)
-				c.setReplicaModeNoLock(address, types.ERR)
-			}
-			// if we still have a good replica, do not return error
-			for _, r := range c.replicas {
-				if r.Mode == types.RW {
-					logrus.Errorf("Ignoring error because %s is mode RW: %v", r.Address, err)
-					err = nil
+			for _, replicaErr := range bErr.Errors {
+				if isSnapshotDiskExist(replicaErr) {
+					snapshotDiskIsExist = true
 					break
```

Comment:
Can we break without checking all replica errors?

Reply:
Hmm...Probably some replicas have `Snapsho Exist` error and some replicas have other errors.
Should no break here.

Reply:
Addressed.  Thank you.